app/sdk/utils/branding/BrandingConfiguration.java

```diff
@@ -0,0 +1,52 @@
+package sdk.utils.branding;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Author: Karis Sponsler
+ * Date: 11/22/17
+ */
+
+public class BrandingConfiguration {
```

Comment:
You should consider adding `@JsonIgnoreProperties` If the response changes it will break this request.

Reply:
addressed.